examples/quickstart.ipynb

```diff
@@ -931,7 +931,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our previous ``read_csv`` example operated on a relatively small dataframe. In the following example, we duplicate the same taxi dataset 100 times and then concatenate them together."
+    "Our previous ``read_csv`` example operated on a relatively small dataframe. In the following example, we duplicate the same taxi dataset 100 times and then concatenate them together.\n",
+    "\n",
+    "Please note that this quickstart notebook is assumed to be run on a machine that is able to perform operations both with pandas and Modin in a single workflow because both pandas and Modin are going to take their own share of memory space. If your machine doesn't have enough resources to execute every cell of the notebook and you see an OOM issue, you will likely want to reduce ``N_copies`` in the cell below."
```

Comment:
It's not very clear what `both with pandas and Modin in a single workflow` means. 

```suggestion
    "Please note that this quickstart notebook is assumed to be run on a machine that is able to perform operations both with pandas and Modin in a single workflow because both pandas and Modin are going to take their own share of memory space. If your machine doesn't have enough resources to execute every cell of the notebook and you see an OOM issue, you most likely need to reduce ``N_copies`` in the cell below."
```

Reply:
You should read the whole sentence. If you have any suggestion, I am happy to add it.

Reply:
I have, suggestion updated.

Reply:
Edited the suggestion, I think it is okay now.

Reply:
yes, thanks!